gostream/ffmpeg/ffmpeg.go

```diff
@@ -0,0 +1,11 @@
+//go:build cgo && !android
+
+// Package ffmpeg is a wrapper around FFmpeg/release6.1.
+// See: https://github.com/FFmpeg/FFmpeg/tree/release/6.1
+package ffmpeg
+
+//#cgo CFLAGS: -I${SRCDIR}/include
+//#cgo linux,arm64 LDFLAGS: -L${SRCDIR}/Linux-aarch64/lib -lavformat -lavcodec -lavutil -lswscale -lm
+//#cgo linux,amd64 LDFLAGS: -L${SRCDIR}/Linux-x86_64/lib -lavformat -lavcodec -lavutil -lswscale -lm
+//#cgo linux,arm LDFLAGS: -L${SRCDIR}/Linux-armv7l/lib -lavformat -lavcodec -lavutil -lm
+import "C"
```

Comment:
I added all of these file to the same package for two reasons:
1. all of the go commands can be centralized here since they hold true at the package level.
2. C types from different packages are of different types so we wouldn't be able to cast a pointer to a C type declared in package A to the same C type declared in package B.

Reply:
Nice! This organization makes a lot of sense to me.